src/cs2fixes.cpp

```diff
@@ -177,8 +197,10 @@ bool CS2Fixes::Load(PluginId id, ISmmAPI *ismm, char *error, size_t maxlen, bool
 
 	ConVar_Register(FCVAR_RELEASE | FCVAR_CLIENT_CAN_EXECUTE | FCVAR_GAMEDLL);
 
-	InitPatches();
-	InitDetours();
+	if (!InitPatches(g_GameConfig))
+		return false;
+	if (!InitDetours(g_GameConfig))
+		return false;
```

Comment:
Do we really have to immediately stop the plugin when any patch or detour fails here? If an update breaks multiple signatures and/or offsets I'd rather see all the failed sigs/offsets at once.

Reply:
good point